apps/air-discount-scheme/backend/src/app/modules/user/user.service.ts

```diff
@@ -27,6 +27,9 @@ export class UserService {
     } = await this.flightService.countThisYearsFlightLegsByNationalId(
       user.nationalId,
     )
+    const availableConnectionFlights = await this.flightService.countThisYearsConnectedFlightsByNationalId(
```

Comment:
Why do we need this, I don't see this used in the frontend and I don't see any reason why the airlines would need this?

Reply:
Ah, that's residual code from an approach that I didn't end up using. Good catch :+1: